src/mobile/package.json

```
@@ -131,7 +131,9 @@
   "resolutions": {
     "js-yaml": ">=3.13.1",
     "handlebars": ">=4.1.2",
-    "set-value": ">=2.0.1"
+    "set-value": ">=2.0.1",
+    "minimist": ">=1.2.2",
+    "acorn": ">=7.1.1"
```

```suggestion
    "mkdirp": "^0.5.3",
    "**/optimist/minimist": "^0.2.1",
    "minimist": "^1.2.5 || >=0.2.1",
    "acorn-globals": ">=6.0.0"
```

I took a closer look at this and I think upgrading `acorn-globals` to 6.0.0 is sufficient since we already upgraded the other instances of `acorn` in #2644 (according to [SNYK-JS-ACORN-559469](https://snyk.io/vuln/SNYK-JS-ACORN-559469), 5.7.4 also contains the fix). We want to avoid breaking other packages that depend on `acorn` if possible.

Similarly, we have three versions of `minimist`: 0.0.8, 0.0.10, and 1.2.0. To avoid breaking packages, we can upgrade `mkdirp` to 0.5.3 (which upgrades `minimist` to 1.2.5), `optimist/minimist` to 0.2.1 (previously 0.0.10), and any other instances of `minimist` to 1.2.5 (previously 1.2.0).

(for these changes to properly take effect in the lockfile, you might need to discard existing changes and then re-run `yarn`)